draft-ietf-sframe-enc.md

```diff
@@ -961,3 +918,87 @@ requirements of the RFC format.  They should be removed before use.
 These test vectors are also available in JSON format at {{TestVectors}}.
 
 {::include test-vectors.md}
+
+# Examples
+
+## RTP
```

Comment:
I would put this above the test vectors appendix, so that it's not buried behind dozens of pages of test vectors.

We should also probably expand this section to have a little more prose, and to cover the "per packet" / "per frame" strategeies.  But we can do that in a follow-on PR.

Reply:
Updated!

Will leave the second part to another PR as you suggest.